contracts/SmartSession.sol

```diff
@@ -251,21 +253,28 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
         /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
         // Check UserOp policies
         // This reverts if policies are violated
-        vd = $userOpPolicies.check({
+        uint256 remainingPoliciesToCheck;
+        (vd, remainingPoliciesToCheck) = $userOpPolicies.check({
```

Comment:
tbh, I called this `p` because it's technically not 'remaining policies to check' , it is 'user op policies checked'